skills_ml/algorithms/embedding/models.py

```diff
@@ -133,3 +138,39 @@ def transform(self, X):
     def fit_transform(self, X, y=None):
         self.fit(X, y)
         return self.transform(X)
+
+
+def visualize_in_tensorboard(model, output_dirname=None, host="127.0.0.1"):
```

Comment:
Does model here specifically mean embedding model?

Reply:
yeah, or specifically gensim keyedvectors class and its descendants.

Reply:
I'll should use some typing here.